gradle/spec/dependabot/gradle/file_updater_spec.rb

```diff
@@ -684,16 +711,25 @@
             )
           end
 
+          before do
+            allow(Dependabot::SharedHelpers).to receive(:run_shell_command)
+          end
+
           its(:content) do
+            expected_command = "gradle --no-daemon --stacktrace wrapper --no-validate-url --gradle-version 9.0.0"
```

Comment:
@gmazzo Another issue I noticed that we may want to address in a follow-up: we're invoking `gradle` instead of `./gradlew`.

This means we're relying on the [system-wide Gradle installation](https://github.com/dependabot/dependabot-core/blob/c938bbf7cb4da88053d4379dcab297a3eaa8c0a7/gradle/Dockerfile#L12), which may not match the version the project expects and could lead to build failures. Using the wrapper avoids this because it automatically downloads the correct Gradle version before applying the wrapper upgrade